### ISSUE TITLE
32bit mmap crash win7

### ISSUE THREAD

**Issue by Kruemmelmonster** [DarthTon/Blackbone#81]
Hey DarthTon

whenever i try to inject into a process via mmap 
the process im trying to inject to crashes
(to be exact it crashes at: Performing Security cookie initialization)

i have a older version of Blackbone wich injects the dll just fine!

my system is Win 7 x64 but the process & injector is 32bit 
```
BlackBone: ManualMap: Mapping image 'MemoryImage_0x00421E30' with flags 0x5002
BlackBone: ManualMap: Loading new image 'MemoryImage_0x00421E30'
BlackBone: ManualMap: Image base allocated at 0x10000000
BlackBone: ManualMap: Performing image copy
BlackBone: ManualMap: Relocating image 'MemoryImage_0x00421E30'
BlackBone: ManualMap: No need for relocation
BlackBone: ManualMap: Loading new dependency 'api-ms-win-crt-math-l1-1-0.dll'
BlackBone: ManualMap: Dependency path resolved to 'C:\Windows\system32\api-ms-win-crt-math-l1-1-0.dll'
BlackBone: ManualMap: Loading new dependency 'ucrtbase.dll'
BlackBone: ManualMap: Dependency path resolved to 'C:\Windows\system32\ucrtbase.dll'
BlackBone: ManualMap: Loading new dependency 'api-ms-win-crt-heap-l1-1-0.dll'
BlackBone: ManualMap: Dependency path resolved to 'C:\Windows\system32\api-ms-win-crt-heap-l1-1-0.dll'
BlackBone: ManualMap: Loading new dependency 'api-ms-win-crt-runtime-l1-1-0.dll'
BlackBone: ManualMap: Dependency path resolved to 'C:\Windows\system32\api-ms-win-crt-runtime-l1-1-0.dll'
BlackBone: ManualMap: Loading new dependency 'msvcp140.dll'
BlackBone: ManualMap: Dependency path resolved to 'C:\Windows\system32\msvcp140.dll'
BlackBone: ManualMap: Performing security cookie initializtion for image 'MemoryImage_0x00421E30'
BlackBone: ManualMap: Performing static TLS initialization for image 'MemoryImage_0x00421E30'
BlackBone: ManualMap: Calling entry point for 'MemoryImage_0x00421E30', Reason:1


```

**Comment by DarthTon** [comment]
Do you remember git revision of the working version?

**Comment by Kruemmelmonster** [comment]
unfortunatly not as i just downloaded it as a zip and didnt clone the repo

but i could pinpoint it down to one function call wich causes the crash: 

```
if (QueueUserAPC( _userCode.ptr<PAPCFUNC>(), _hWorkThd.handle(), _userCode.ptr<ULONG_PTR>() ))
    {
```

so i guess its related to #71

**Comment by DarthTon** [comment]
May be fixed in 172fd60cd1afbf17daee0e70e85a46a58ca253ca.

**Comment by Kruemmelmonster** [comment]
this seems to have fixed the issue! im so thankful! if you have a donation link i would be happy to donate for some beer ;)
